### PR TITLE
v6 release notes

### PR DESCRIPTION
**Pull request by kbrock**
updates for the next version.

This version is mostly for rails6.1 functionality

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,13 @@ a nice looking [Changelog](http://keepachangelog.com).
 
 ## Version [Unreleased] <small>tbd</small>
 
+## Version [4.0.0] <small>2022-01-26</small>
+
+* Added Rails 6.2 support
```
Comment (Fryguy) [inline CHANGELOG.md:10]: ?
```suggestion
* Added Rails 6.1 support
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,29 +1,37 @@
...
-Doing our best at supporting [SemVer](http://semver.org/) with
-a nice looking [Changelog](http://keepachangelog.com).
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
 
-## Version [Unreleased] <small>tbd</small>
+## [Unreleased]
 
-## Version [3.0.0] <small>2020-09-28</small>
+## [6.1.0] - 2022-01-26
```
Comment (chessbyte) [inline CHANGELOG.md:8]: why does PR title say v4 when this is 6.1?

**Comment by kbrock** [comment]
updates:

- fixed changelog formatting
- changed ci to be rails 60 and 61 (surprised it was 5.2 and 6.0)
- version is now 6.0

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,29 +1,37 @@
-# Virtual Attributes Changelog
+# Change Log
 
-Doing our best at supporting [SemVer](http://semver.org/) with
-a nice looking [Changelog](http://keepachangelog.com).
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
```
Comment (Fryguy) [inline CHANGELOG.md:4]: This line should now be changed that we are _NOT_ following symantic version, but instead matching Rails' release version numbers.

**Comment by kbrock** [comment]
aah. rolled back rails 61 changes - that PR has not been merged yet

File: CHANGELOG.md [hunk 4]
```diff
@@ -32,47 +40,48 @@ a nice looking [Changelog](http://keepachangelog.com).
...
 * Added support for Rails 5.1
 
-## Version 0.1.0 <small>2019-01-17</small>
+## 0.1.0 - 2019-01-17
 
 * Initial Release
 * Extracted from ManageIQ/manageiq
 
 [Unreleased]: https://github.com/ManageIQ/activerecord-virtual_attributes/compare/v3.0.0...HEAD
+[4.0.0]: https://github.com/ManageIQ/activerecord-virtual_attributes/compare/v3.0.0...v4.0.0
```
Comment (Fryguy) [inline CHANGELOG.md:84]:
```suggestion
[Unreleased]: https://github.com/ManageIQ/activerecord-virtual_attributes/compare/v6.0.0...HEAD
[6.0.0]: https://github.com/ManageIQ/activerecord-virtual_attributes/compare/v3.0.0...v6.0.0
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,29 +1,37 @@
...
+This project adheres to [Semantic Versioning](http://semver.org/).
 
-## Version [Unreleased] <small>tbd</small>
+## [Unreleased]
 
-## Version [3.0.0] <small>2020-09-28</small>
+## [6.0.0] - 2022-02-02
+
+* Added Rails 6.1 support
+* Dropped support for Rails 5.0, 5.1, 5.2
```
Comment (Fryguy) [inline CHANGELOG.md:11]:
```suggestion
* **BREAKING** Dropped support for Rails 5.0, 5.1, 5.2
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,29 +1,37 @@
...
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
 
-## Version [Unreleased] <small>tbd</small>
+## [Unreleased]
 
-## Version [3.0.0] <small>2020-09-28</small>
+## [6.0.0] - 2022-02-02
+
+* Added Rails 6.1 support
```
Comment (Fryguy) [inline CHANGELOG.md:10]:
```suggestion
* **BREAKING** This gem will now no longer follow Symantic Versioning, but instead follow Rails' versioning numbers in order to simplify version matches between them both.
* Added Rails 6.1 support
```

File: CHANGELOG.md (hunk 4 above)
Comment (kbrock) [reply CHANGELOG.md:84]: ugh.

File: CHANGELOG.md (hunk 6 above)
Comment (kbrock) [reply CHANGELOG.md:10]: yea. as I added Breaking, it looks better at the top

Comment (Fryguy) [reply CHANGELOG.md:10]: Oh yeah..agreed.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,29 +1,40 @@
-# Virtual Attributes Changelog
+# Change Log
 
-Doing our best at supporting [SemVer](http://semver.org/) with
-a nice looking [Changelog](http://keepachangelog.com).
+Versioning is now following active record versioning. And not supporting SemVer.
```
Comment (Fryguy) [inline CHANGELOG.md:3]: The wording of this statement implies a point in time, however this is an ephemeral statement.  That is, "now" won't make sense 5 versions in the future.  We should have the point-in-time statement in the versioned part as I commented in https://github.com/ManageIQ/activerecord-virtual_attributes/pull/104/files#r797922371.  Then, here, we can say something like:

```suggestion
Versioning of this gem follows ActiveRecord versioning, and does not support SemVer.
```

File: CHANGELOG.md (hunk 6 above)
Comment (Fryguy) [reply CHANGELOG.md:14]: I think it should be in both places, so we know _when_ this versioning change was made...also commented in the statement at the top.

**Comment by kbrock** [comment]
update:

- more wording changes in CHANGELOG around semver and breaking changes

**Pull request merged**